src/redux/sagas/customerTypeSaga.tsx

```diff
@@ -57,6 +58,20 @@ const putCustomerType = async (payload: ICustomerType, id: number) => {
   return data;
 };
 
+const deleteCustomerType = async (id: number) => {
+  const accessToken = JSON.parse(localStorage.getItem(sessionToken) as string).access;
+  const { data } = await apiClient.delete(
+    `customer_type/${id}/`,
```

Comment:
The string `customer_type/` is using several time, so maybe better to use it above the code 
`const costometTypeUrl = 'customer_type/' ` :)